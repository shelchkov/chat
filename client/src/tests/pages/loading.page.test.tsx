import React from "react"
import { shallow } from "enzyme"

import { LoadingPage } from "../../pages/loading.page"

describe("loading page", (): void => {
	it("should render correctly", (): void => {
		const loadingPage = shallow(<LoadingPage />)
		expect(loadingPage).toMatchSnapshot()
	})
})
