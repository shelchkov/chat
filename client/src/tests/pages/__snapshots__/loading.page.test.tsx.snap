// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading page should render correctly 1`] = `ShallowWrapper {}`;
